components/sbm-recipes-boot-upgrade/src/main/java/org/springframework/sbm/boot/upgrade_27_30/conditions/LoggingDateFormatCondition.java

```diff
@@ -0,0 +1,20 @@
+package org.springframework.sbm.boot.upgrade_27_30.conditions;
+
+import org.springframework.sbm.boot.upgrade_27_30.filter.LoggingDateFormatPropertyFinder;
+import org.springframework.sbm.engine.context.ProjectContext;
+import org.springframework.sbm.engine.recipe.Condition;
+
+import java.util.List;
+
+public class LoggingDateFormatCondition implements Condition {
+
+    @Override
+    public String getDescription() {
+        return null;
```

Comment:
Could you add a description "Check if 'logging.pattern.dateformat' is declared."